src/cloud/docker/docker-extend.md

```diff
@@ -143,9 +187,96 @@ To add a new PHP extension:
 
 1. Test the extension by specifying the [Docker build sources].
 
-[multiple compose files]: https://docs.docker.com/compose/reference/overview/#specifying-multiple-compose-files
-[service versions]: https://devdocs.magento.com/cloud/docker/docker-containers.html#service-containers
+### PHP extension configuration reference
+
+Use the following attributes to specify the PHP extension configuration in the getConfig method located in the [ExtensionResolver.php] file. The configuration you specify depends the extension installation method: from the official Docker PHP images, from the PECL repository, or using an installation script.
+
+| Configuration option | Description
+| -------------------- | ------------
+| PHP version constraint | Specifies the extension versions to install. If different versions have different installation requirements, you must add the configuration for each version.
+| `EXTENSION_TYPE_CORE` | Extension that can be installed from a `docker-php-source` image. See [PHP, Docker Official Images] in Docker Hub.
+| `EXTENSION_TYPE_PECL` | Extensions that can be installed from the [PECL] repository. See _PECL extensions_ in the [How to install more PHP extensions][PHP, Docker Official Images] documentation in Docker Hub.
```

Comment:
The first thing you see is the PHP heading on that Docker page. Without a useful anchor, it may need to guide readers more carefully. The following suggestion is similar to the next option, too.

```suggestion
| `EXTENSION_TYPE_PECL` | Extensions that can be installed from the [PECL] repository. See **PECL extensions** in the _How to install more PHP extensions_ section of the [PHP, Docker Official Images] page in Docker Hub.
```